conf/app.routes

```diff
@@ -363,7 +363,7 @@ GET         /:srn/change-are-repayments-instalments/:index
 POST        /:srn/change-are-repayments-instalments                                                            controllers.nonsipp.loansmadeoroutstanding.AreRepaymentsInstalmentsController.onSubmit(srn: Srn, index: Max5000 = 1, mode: Mode = CheckMode)
 POST        /:srn/change-are-repayments-instalments/:index                                                     controllers.nonsipp.loansmadeoroutstanding.AreRepaymentsInstalmentsController.onSubmit(srn: Srn, index: Max5000, mode: Mode = CheckMode)
 
-GET         /:srn/interest-on-loan                                                                             controllers.nonsipp.loansmadeoroutstanding.InterestOnLoanController.onPageLoad(srn: Srn, index: Max5000 = 1, mode: Mode = NormalMode)
+GET         /:srn/                                                                             controllers.nonsipp.loansmadeoroutstanding.InterestOnLoanController.onPageLoad(srn: Srn, index: Max5000 = 1, mode: Mode = NormalMode)
```

Comment:
hi @Arei1997, Is this change done intentionally ?

Reply:
Nope, let me fix that not sure why that happened

Reply:
Should be OK now

Reply:
I reverted the change on /:srn/interest-on-loan endpoint in app.routes